plugins/inputs/slurm/sample.conf

```diff
@@ -0,0 +1,28 @@
+# Gather SLURM metrics
+[[inputs.slurm]]
+  ## Slurmrestd URL. Both http and https can be used as schemas.
+  url = "http://127.0.0.1:6820"
+
+  ## Credentials for JWT-based authentication.
+  # username = "foo"
+  # token = "topSecret"
+
+  ## Ignored endpoints
+  ## List of endpoints a user can ignore, choose from: diag, jobs,
+  ##   nodes, partitions, reservations
+  ## Please note incorrect endpoints will be silently ignore and
+  ## that endpoint names are case insensitive.
+  # ignored_endpoints = []
```

Comment:
Wouldn't it make more sense to specify what should be collected? It's easier to read and easier code-wise...

Reply:
The idea was to leverage the zero value for slices so that the 'zeroconf' would be to collect everything. However, if you'd rather have the opposite behaviour I would love to swap things around...

Reply:
I think having it swapped is better. Just set the corresponding endpoints in `Init()` if the list is empty and be done. This makes it also easier to extend the list and it will show the user what is actually collected.

Reply:
After adding the changes I can't but agree with your view 😜. These swap from disabling to enabling inputs has been applied on f943d9d6bfe883fdc425170ec10c4c3193211d23.